### PR TITLE
Chore: Add descriptions for light and disabled tokens

### PR DESCRIPTION
**Pull request by alexandra-lim**
Adds the descriptions for our light and disabled colour tokens

### CODE REVIEW

File: src/transformed/transformed-light.json [hunk 1]
```diff
@@ -427,32 +427,39 @@
...
         "value": "#F5F7FA",
-        "type": "color"
+        "type": "color",
+        "description": "Used as the background colour for disabled components"
       },
       "nakedContent": {
         "value": "#F5F7FA",
-        "type": "color"
+        "type": "color",
+        "description": "Used for content in disabled components with little to no styling"
```
Comment (alexandra-lim) [inline src/transformed/transformed-light.json:441]: I'm sure there's a better way to describe nakedContent other than "little or no styling". We should also re-evaluate the need for this token, it's only being used in the Pill as a background color: https://git.fullscript.io/developers/hw-admin/-/blob/staging/app/javascript/aviary/components/Pill/Pill.styles.ts#L79

Comment (iryanclarke) [reply src/transformed/transformed-light.json:441]: I think that makes sense!

**Review by iryanclarke — APPROVED** [review]

File: src/transformed/transformed-light.json (hunk 1 above)
Comment (CCAyl) [reply src/transformed/transformed-light.json:441]: good catch - maybe we can decide if we need this token during our meeting today?

**Review by CCAyl — APPROVED** [review]

File: src/transformed/transformed-light.json (hunk 1 above)
Comment (alexandra-lim) [reply src/transformed/transformed-light.json:441]: I've gone ahead and removed the desc from nakedContent, will open a follow up MR to remove it completely

**Pull request merged**